MAUI/DataGrid/Editing.md

```diff
@@ -0,0 +1,376 @@
+# Editing in MAUI DataGrid (SfDataGrid)
+
+The [Syncfusion .NET MAUI DataGrid]() (SfDataGrid) control supports editing cell values by setting the [SfDataGrid.AllowEditing property](), [SfDataGrid.NavigationMode]() to Cell, and setting the [SfDataGrid.SelectionMode]() to any value other than None.
+
+To enable editing, follow the code example:
+
+{% tabs %}
+{% highlight xaml %}
+    <syncfusion:SfDataGrid x:Name="dataGrid"
+                           AllowEditing="True"
+                           SelectionMode="Multiple"    
+                           NavigationMode="Cell" 
+                           ItemsSource="{Binding Orders}" />
+{% endhighlight %}
+{% highlight c# %}
+dataGrid.AllowEditing = true;
+dataGrid.SelectionMode = SelectionMode.Multiple;
+dataGrid.NavigationMode = NavigationMode.Cell;
+{% endhighlight %}
+{% endtabs %}
+
+![DataGrid with editing](Images\editing\maui-datagrid-editing.png)
+
+## Column editing
+
+To enable or disable editing for a specific column, you need to set the [DataGridColumn.AllowEditing]() property.
+
+{% tabs %}
+{% highlight xaml %}
+    <syncfusion:DataGridTextColumn AllowEditing="True"
+                                   MappingName="OrderID" />
+{% endhighlight %}
+{% highlight c# %}
+DataGridTextColumn column = new DataGridTextColumn();
+column.MappingName="OrderID";
+column.AllowEditing = false;
+{% endhighlight %}
+{% endtabs %}
+
+N>The `DataGridColumn.AllowEditing` takes higher priority than the `SfDataGrid.AllowEditing`.
+
+## Entering into edit mode
+
+To enter edit mode by simply tapping or double-tapping the grid cells, set the [SfDataGrid.EditTapAction]() property.
+
+{% tabs %}
+{% highlight xaml %}
+    <sfgrid:SfDataGrid x:Name="dataGrid"
+                               AllowEditing="True"
+                               EditTapAction="OnTap">
+{% endhighlight %}
+{% highlight c# %}
+//Enter edit mode in single tap
+this.dataGrid.EditTapAction = TapAction.OnTap;
+//Enter edit mode in double tap
+this.dataGrid.EditTapAction = TapAction.OnDoubleTap;
+{% endhighlight %}
+{% endtabs %}
+
+N> The keyboard will be collapsed when editing grid cell gets unfocused.
```

Comment:
The keyboard will collapse when the editing grid cell loses focus

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/113881396/1c5582d0-1055-4658-b19b-2cd8dc878a93)